awscli/examples/networkflowmonitor/stop-query-workload-insights-top-contributors.rst

```diff
@@ -0,0 +1,9 @@
+**To stop a query**
+
+The following ``stop-query-workload-insights-top-contributors`` example stops the query in the specified account. ::
+
+    aws networkflowmonitor stop-query-workload-insights-top-contributors \ 
+        --scope-id e21cda79-30a0-4c12-9299-d8629d76d8cf \
+        --query-id 1fc423d3-b144-37a6-80e6-e2c7d26eea0c
+
+This command produces no output.
```

Comment:
Every example needs a for more information link.
https://aws.github.io/aws-cli/docs_styleguide.html#for-more-information-link